DEMO.md

```diff
@@ -0,0 +1,104 @@
+(NOTE: This document is still being updated)
+
+
+# ATD-AVD Dual DataCenters Lab Guide
+
+## Fabric Build Detailed Instructions:
+
+### Step 1:
+
+Open the link provided by Arista representative during registration. The provided link should provide a page similar to the image below.
+
+
+![](./images/Start_Screen.png)
+
+
+Check and make sure that your topology is "Running". If not, pls. go ahead and click "START" to bring up the lab.
+
+Note: Pls. take note of the Lab Credentials and Passwords, these are necessary to move to the next steps.
+
+### Step 2:
+
+Once the topology is running, you should be able to access your lab by clicking on "Click Here to Access Topology"
+
+### Step 3:
+
+Open your VScode by clicking "Programmability IDE" shown on the image below.
+
+Enter the Programmability IDE password when prompted.
+
+
+![](./images/Programmability_IDE.png)
+
+
+Using PWD command, check and make sure that you are inside the proper directory. If not, type "cd labfiles/AristaValidatedDesigns" to get into the correct directory.
```

Comment:
Need to add a step here on how to open up the terminal window, by default it is not shown on a new window.  User will not know where to put the commands below (Lines 37-41)